src/useFormState.test.tsx

```diff
@@ -228,7 +228,7 @@ describe("useFormState", () => {
               <button data-testid="makeLocalChanges" onClick={makeLocalChanges} />
               <button data-testid="refreshData" onClick={() => setData(data2)} />
               <button data-testid="saveData" onClick={() => setData(data3)} />
-              <div data-testid="changedValue">{JSON.stringify(form.changedValue)}</div>
```

Comment:
To keep this test case's scenario the same, I had to remove the "mostly for extra assertions" `changedValue` usage, since now the `.changedValue` call has the side-effect of making form-state think "we sent this value to the server".

Which is fine, but not the scenario this test wanted to test.